### PR TITLE
[PLATFORM-313] Visual editor docs content

### PR DESCRIPTION
**Pull request by fonty1**
Visual editor content injection.
/docs/visual-editor content. More a less a copy paste of https://www.streamr.com/help/userGuide

** exporting `CreateStreamButton` because I can and it's a better than a picture of the button. 
** May contain intentional traces of other docs pages

### CODE REVIEW

File: app/src/docs/components/VisualEditorPage/Content.mdx [hunk 1]
```diff
@@ -1,3 +1,637 @@
...
+import AddTwitterStream from './images/canvases/addTwitterStream.png'
+import ModuleBrowser from './images/canvases/moduleBrowser.png'
+import ConnectingStreamToModule from './images/canvases/connectingStreamToModule.png'
+import StartHistoricalRun from './images/canvases/startHistoricalRun.png'
+import PlaybackOptions from './images/canvases/playbackOptions.png'
+import StartRealtimeRun from './images/canvases/startRealtimeRun.png'
+import StopRealtimeRun from './images/canvases/stopRealtimeRun.png'
+
+import JavaModuleOnCanvas from './images/extensions/javaModuleOnCanvas.png'
+import JavaModuleExampleOnCanvas from './images/extensions/javaModuleExampleOnCanvas.png'
```
Comment (juhah) [inline app/src/docs/components/VisualEditorPage/Content.mdx]: The actual file name begins with capital `J`, got an error for that while building.

File: app/src/docs/components/VisualEditorPage/codeSnippets.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
+export const JavaModule =
```
Comment (juhah) [inline app/src/docs/components/VisualEditorPage/codeSnippets.js]: The snippets would look better without the start and end newlines in my opinion... Or is it designed like this?

**Comment by fonty1** [comment]
> I just skimmed the actual content but visually everything looks good 😄

Yep, no worries, content review is not required at this stage as there'll be future passes at the content once the Editor is in a state to take screenshots of it, etc.

File: app/src/docs/components/VisualEditorPage/codeSnippets.js (hunk 2 above)
Comment (fonty1) [reply app/src/docs/components/VisualEditorPage/codeSnippets.js]: hmm, I'll take another look and perhaps add the line numbers back in while I'm at it. 

https://share.goabstract.com/3fe2aa03-3291-425f-ad69-4fb0e191a670 - shows some appetite for top/bottom margin at least.

Comment (juhah) [reply app/src/docs/components/VisualEditorPage/codeSnippets.js]: Yep, lots of padding :) You could get rid of the extra newlines and just use padding.

Comment (fonty1) [reply app/src/docs/components/VisualEditorPage/codeSnippets.js]: sounds good 👍

**Pull request merged**